hack/podman-registry-go/registry_test.go

```diff
@@ -0,0 +1,31 @@
+package registry
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestStartAndStopMultipleRegistries(t *testing.T) {
+	binary = "../podman-registry"
+
+	registries := []*Registry{}
+
+	// Start registries.
+	for i := 0; i < 3; i++ {
+		reg, err := Start()
+		require.NoError(t, err)
+		assert.True(t, len(reg.Image) > 0)
+		assert.True(t, len(reg.User) > 0)
+		assert.True(t, len(reg.Password) > 0)
+		assert.True(t, len(reg.Port) > 0)
+		registries = append(registries, reg)
```

Comment:
If you do add a teardown as suggested below, it might make sense to reorder the above as:
````
    assert(Port)
    append(reg)
    assert(all the rest)
````
That way `teardown` will have the best possible `registries` array early on.

Reply:
Accounted for that with a multi error :+1: